guide/12-enrich-data-with-thematic-information/part1_introduction_to_geoenrichment.ipynb

```diff
@@ -11,139 +11,576 @@
     "# Part 1 - Introduction to GeoEnrichment"
```

Comment:
<h3>Discover Demographic Variables </h3><p>First, we can discover the variables available with the <code>enrich_variables</code> property of  the <code>Country</code> object.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>